src/_includes/tags.html

```diff
@@ -31,7 +31,7 @@
 
 {% macro render(tags, path, hide_heading=false, display_block=false) %}
 <div class="tags {{'tags__hide-heading' if hide_heading}} {{'tags__block-list' if display_block}}">
-    <h1 class="tags_heading {{'u-visually-hidden' if hide_heading}}">Topics:</h1>
+    <span class="tags_heading {{'u-visually-hidden' if hide_heading}}">Topics:</span>
```

Comment:
Same deal as the share module. https://github.com/cfpb/cfgov-refresh/pull/972/files#r39281082